jobbergate-api/jobbergateapi2/main.py

```diff
@@ -43,14 +48,10 @@ async def init_database():
     await storage.database.connect()
 
 
-@subapp.on_event("shutdown")
+@app.on_event("shutdown")
 async def disconnect_database():
     """
     Disconnect the database
     """
     logger.debug("Disconnecting database")
     await storage.database.disconnect()
```

Comment:
there is a specific PR for this change: https://github.com/omnivector-solutions/jobbergate/pull/40. Should we close that one or rollback here?

Reply:
I think it's better to rollback here since the PR is specific for the cli feature, right?

Reply:
yes, will do that.

Reply:
revised in latest commit.